api/helpers/db.js

```diff
@@ -29,24 +30,7 @@ try {
 }
 let db = pgp(config);
 
-let dbtagkeys;
-// let dbcasekeys;
-
-async function initKeys() {
-  // we're just getting the keys for validation, which are the same for
-  // every language, so hard-coding 'en' here is OK.
-  // dbcasekeys = (await db.one(`
-  //   SELECT to_json(array_agg(key)) AS keys
-  //   FROM localized_case_field_values
-  //   WHERE language = 'en';
-  // `)).keys;
-  dbtagkeys = (await db.one(`
-    SELECT to_json(array_agg(key)) as keys
-    FROM rotate_tags_localized('en') AS tagvalues
-    WHERE tagvalues.key <> 'language';
-  `)).keys;
-}
-initKeys().then(() => console.log("keys initialized")); // we'll need these when users submit data
+const dbtagkeys = JSON.parse(fs.readFileSync("api/helpers/data/tagkeys.json"));
```

Comment:
we could use the `shared-field-options` file i'm using for the front end for this so we don't duplicate the list of tags in the code base.
```
const sharedFieldOptions = require("../helpers/shared-field-options.js");
const tagKeys = sharedFieldOptions.tags.map(tag => tag.key);
```

Reply:
Yes. I will go over all of these in one go when we merge your localization branch.